tests/foreman/ui/test_subnet.py

```diff
@@ -110,12 +110,10 @@ def test_positive_add_domain(self):
                 (strategy2, value2 % domain.name))
             # Depending upon the number of domains either, checkbox or
             # selection list appears.
-            if element:
-                self.assertIsNotNone(element)
-            elif checkbox_element:
+            if element is None and checkbox_element is None:
+                raise UIError('Neither checkbox or select list is present')
+            if checkbox_element:
                 self.assertTrue(checkbox_element.is_selected())
-            else:
-                self.assertIsNotNone()
```

Comment:
Feeling uncomfortable of test not asserting anything in case we have selection list, but it looks like there's really no use of any assertion...